README.md

```diff
@@ -173,6 +173,17 @@ of the next Istio release, and `previous_version` to be the version of the previ
 
 ### Creating a patch release
 
+A few days before the patch release, the release managers should notify the Doc WG that the release
+is built and is starting it's long running qualification test. At this time, move the doc automation
+tests to use the new release to verify automated doc testing passes. To move to a new release
+(make sure you are in the patch's release branch):
+
+1. `go get istio.io/istio@<1.X.Y>`
```

Comment:
This `<1.X.Y>` seems confusing. It's supposed to be `go get istio.io/istio@1.6.1`, not `go get istio.io/istio@<1.6.1>` right?
```suggestion
1. `go get istio.io/istio@1.X.Y`
```